CRM/Utils/Check/Component/Env.php

```diff
@@ -699,7 +704,7 @@ public function checkExtensions() {
         $message = ts('All extensions are up-to-date:');
       }
       $messages[] = new CRM_Utils_Check_Message(
-        'extensionsOk',
+        __FUNCTION__ . 'Ok',
```

Comment:
These message names were non-standard and really needed to be changed anyway. The assumption with status checks has always been that they start with the name of the function. And that's especially important now that we're allowing filtering by name.

Reply:
I agree in general. I'm not certain why the message names are different when the extensions are OK versus when they need updates.  The result is that someone hushing `extensionUpdates` at the level `warning` would still see the `extensionsOk` message when things are OK.  Maybe that's good, but I think it might be a surprise.

But speaking of surprises, the only issue with changing message names is that existing statusPreference entries will no longer apply to anything.  This needs an upgrade step to take any statusPreferences for `extensionsOk` and edit them to be `checkExtensionsOk` and so forth for the others that are edited.

Reply:
Yep, I'll add that upgrade SQL.

@agh1 the reason we have to append strings to the end of `__FUNCTION__` is that this function potentially generates more than one message at once, and names must be unique.

Reply:
Added the upgrade bit.